cmd/documentation/exchanges_templates/alert.tmpl

```diff
@@ -0,0 +1,61 @@
+{{define "exchanges alert" -}}
+{{template "header" .}}
+## Alert
+
++ This package allows for multiple routines to wait for a state change on any required data.
```

Comment:
Please update this with more serious example code. Add in writing that this package uses a pool of channels for alerts and then add in writing its the reason to not capture the returning channel because it can lead to false alerts as a result